department-of-veterans-affairs/va.gov-cms
# VACMS-1600 disable workflow participants

## Description

See #1600.

## Testing done

* Edit a currently published node (http://va-gov-cms.lndo.site/node/3004/edit)
  * Verify the Participants tab is no longer visible.
* Change the state of the node to draft.
* Take the new draft through the workflow.  Errors should not occur.
* Look in watchdog for any new errors/notices.

This PR replaced #1987 due to issues with devshop.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/department-of-veterans-affairs/va.gov-cms/1998)
